configs/templates/ufs-weather-model/spack.yaml

```diff
@@ -15,5 +15,4 @@ spack:
   include: []
 
   specs:
-    - ufs-weather-model-env~debug
-    - ufs-weather-model-env+debug
+    - ufs-weather-model-env~debug~python~shared
```

Comment:
So you don't want the debug version?

Reply:
Not as a default, though in fairness I don't know how commonly used this is. I lean toward leaving in the option but not enabling by default.

Reply:
The ufs-weather-model requires both the release and debug version for the regression testing.